Cabal/Cabal.cabal

```diff
@@ -263,7 +263,9 @@ library
       Distribution.Compat.Binary.Generic
 
   default-language: Haskell98
-  default-extensions: CPP
+  -- starting with GHC 7.0, rely on {-# LANGUAGE CPP #-} instead
+  if !impl(ghc >= 7.0)
```

Comment:
The main reason I write `!impl(ghc >= 7.0)` is that I'm not sure what the semantics are if the `impl(ghc)` is false. If in fact those two variants are equivalent, there's no reason to use the `!`-version :-)